assets/i18n/en.json

```diff
@@ -501,6 +501,9 @@
     "I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": "Oppia lets you <a href=\"/creator_dashboard?mode=create\">create</a> and <a href=\"/library\">share</a> explorations on a wide range of subjects, limited only by your imagination.",
     "I18N_SPLASH_TITLE": "Think outside the books.",
     "I18N_START_HERE": "Click here to start!",
+    "I18N_STORY_VIEWER_COMPLETED_CHAPTER": "<[title]> - Completed!",
+    "I18N_STORY_VIEWER_LOCKED_CHAPTER": "<[title]> - LOCKED",
+    "I18N_STORY_VIEWER_PENDING_CHAPTER": "<[title]>",
```

Comment:
I think let's drop this field altogether. It will confuse the translators because there is nothing to translate.

Reply:
Done